Prevent 'makepkg' showing up in version string.

This is because the branch name is acquired during creation of
the version string. The branch variable is normally empty, as
shown by the diff below.

--- a/cmake/make_versioncpp.py
+++ b/cmake/make_versioncpp.py
@@ -133,11 +133,6 @@ branch = ""
 build_no = INVALID_BUILD_NO

 try:
-    branch = check_output(['git', 'rev-parse', '--abbrev-ref', 'HEAD'], universal_newlines=True).strip()
-    if (branch == "master") or (branch[:7] == "release"):
-        branch = ""
-    else:
-        branch += " "
     commit = check_output(["git", "show", "--no-show-signature", "-s", "--format=%h", "--abbrev=7", "HEAD"], universal_newlines=True).strip()
     timestamp = float(check_output(["git", "show", "--no-show-signature", "-s", "--format=%ct", "HEAD"], universal_newlines=True).strip())
     build_no = ".".join([datetime.utcfromtimestamp(timestamp).strftime("%Y-%m-%d"), commit])

